.azure/pipelines/stages/publish-symbols.yml

```diff
@@ -0,0 +1,67 @@
+# publish-symbols.yml
+# Description: Defines a stage for publishing symbols to SymWeb and the Microsoft symbol server (MSDL)
+# Parameters:
+#   artifacts: [ string ]
+#     Names of artifacts containing symbol packages of interest
+#   condition: string
+#     A condition which can be used to skip the job completely
+#   dependsOn: string | [ string ]
+#     For fan-out/fan-in. See
+#     https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#dependencies
+#   jobDisplayName: string
+#     Friendly name of the single job in this stage to display in the UI. Defaults to "Publish symbols".
+#   stageDisplayName: string
+#     Friendly stage name to display in the UI. Defaults to "Publish symbols stage".
+#   stageDisplayName: string
+#     Name of this stage. Defaults to the empty string. Useful when other stages depend on this one. See
+#     https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#dependencies
+
+parameters:
+  artifacts: []
+  condition: ''
+  dependsOn: ''
+  jobDisplayName: Publish symbols
+  stageDisplayName: Publish symbols stage
+  stageName: ''
+
+stages:
+- stage: ${{ parameters.stageName }}
+  condition: ${{ parameters.condition }}
+  dependsOn: ${{ parameters.dependsOn }}
+  displayName: ${{ parameters.stageDisplayName }}
+  jobs:
+  - template: ../jobs/default-build.yml
+    parameters:
+      agentOs: Windows
+      codeSign: false
+      installJdk: false
+      installNodeJs: false
+      isTestingJob: false
+      jobDisplayName: ${{ parameters.jobDisplayName }}
+      jobName: '_'
```

Comment:
`jobName` is not useful here because no other jobs can depend on this one.

Reply:
This affects the UI as well. Can we set it to something meaningful?

Reply:
`jobName` doesn't matter when `jobDisplayName` is set